configure

```diff
@@ -731,11 +731,14 @@ def configure_library(lib, output):
     if pkg_cflags:
       output['include_dirs'] += (
           filter(None, map(str.strip, pkg_cflags.split('-I'))))
+    elif options.__dict__[shared_lib + '_includes']:
```

Comment:
I'm inclined to think the `__dict__` approach is correct her because it skips over empty flags.